### DISCUSSION TITLE
[BUG] salt-cloud fix doc on tmp folder info, and FR to disable use of uuid in tmp folder

### DISCUSSION THREAD

**Discussion by amalaguti** [saltstack/salt#67331] [Feature Requests]
**Description**
This would be a documentation fix plus feature request 
Based on Cloud docs here https://docs.saltstack.com/en/latest/topics/cloud/troubleshooting.html#tmp-is-mounted-as-noexec
The tmp folder used to deploy minion on the instance should be /tmp/.saltcloud

But it's not correct, it's /tmp/.saltcloud-(random uuid)
https://github.com/saltstack/salt/blob/5281b622e452f8449132762ebe2aa2875a277930/salt/utils/cloud.py#L1446
tmp_dir = "{0}-{1}".format(tmp_dir.rstrip("/"), uuid.uuid4())

Should the docs be updated?

As a feature request, would be nice if this was set using an option, by default enabled, but given the change to the user to not append the uuid.

**Setup**
Salt: 3001.1

**Steps to Reproduce the behavior**
Deploy an instance, in my case using AWS EC2 as provider.
use:
salt-cloud -p profile instance_name --keep-tmp

**Expected behavior**
1. Update documentation to reflect the right information
2. Offer the option to disable the use of uuid

**Comment by barbaricyawps** [comment]
Just adding my two cents that these should be two separate issues: one for the doc request and one for the feature request.
